index.html

```diff
@@ -3158,7 +3158,10 @@ <h2 id="AudioDestinationNode">
           Giving a concrete example, if the audio hardware supports 8-channel
           output, then we may set <a href=
           "#widl-AudioNode-channelCount">channelCount</a> to 8, and render 8
-          channels of output.
+          channels of output. <span class="synchronous">An
+          <code>NotSupportedError</code> MUST be fired if the implementation
+          does not support changing the <a href=
+          "#widl-AudioNode-channelCount">channelCount</a></span>.
```

Comment:
Quoting the current spec:

>  maxChannelCount is the maximum number of channels that this hardware is capable 
> of supporting

The hardware can supports more than two, Gecko does not right now. `maxChannelCount` reports the hardware count, not what is available. Of course we can change this, but as it is right now, it's not illegal per spec.

I think it would be more useful to report the number of channels of the hardware on `maxChannelCount` and throw with `NotSupportedError` when setting the `channelCount` to something that the engine does not support, or `IndexSizeError` when it's outside [1, `maxChannelCount`].

I don't feel strongly about this though, a number of solutions are acceptable.